frontend/public/components/nav.jsx

```diff
@@ -338,6 +338,7 @@ export class Nav extends React.Component {
           <NavSection text="Troubleshooting" icon="fa-life-ring">
             <HrefLink href="/search" name="Search" onClick={this.close} startsWith={searchStartsWith} />
             <ResourceNSLink resource="events" name="Events" onClick={this.close} />
+            { /* FIXME: These links are wrong for OpenShift. */ }
```

Comment:
Should we open a story for that ?

Reply:
Yeah, I've been using CONSOLE-585 to track getting metrics to work in OpenShift generally. Will add to that story. There are a few other related problems.

Reply:
I ended up opening a separate issue for this so we don't lose track (CONSOLE-602)